README.md

```diff
@@ -1,75 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly update your readme file

-You can use this [template](https://github.com/microverseinc/readme-template) 
and make sure there are no broken links(screenshot, license, etc.) and empty sections. 
You should change the readme template's sentence with your sentence.